### ISSUE TITLE
Compilation error: use of deleted function

### ISSUE THREAD

**Issue by itsosmx** [JiriBilek/ESP_ATMod#29]
```err
C:\Users\husse\Downloads\esp01-firmware-main\ESP_ATMod-master\ESP_ATMod\command.cpp: In function 'void cmd_AT_CIPSSLCERT()':
C:\Users\husse\Downloads\esp01-firmware-main\ESP_ATMod-master\ESP_ATMod\command.cpp:2950:32: error: use of deleted function 'BearSSL::X509List& BearSSL::X509List::operator=(const BearSSL::X509List&)'
 2950 |     CAcert = BearSSL::X509List();
      |                                ^
In file included from C:\Users\husse\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.1.2\libraries\ESP8266WiFi\src/WiFiClientSecureBearSSL.h:29,
                 from C:\Users\husse\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.1.2\libraries\ESP8266WiFi\src/WiFiClientSecure.h:23,
                 from C:\Users\husse\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.1.2\libraries\ESP8266WiFi\src/WiFiServerSecure.h:20,
                 from C:\Users\husse\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.1.2\libraries\ESP8266WiFi\src/ESP8266WiFi.h:41,
                 from C:\Users\husse\Downloads\esp01-firmware-main\ESP_ATMod-master\ESP_ATMod\command.cpp:23:
C:\Users\husse\AppData\Local\Arduino15\packages\esp8266\hardware\esp8266\3.1.2\libraries\ESP8266WiFi\src/BearSSLHelpers.h:127:15: note: declared here
  127 |     X509List& operator=(const X509List& that) = delete;
      |               ^~~~~~~~

exit status 1

Compilation error: use of deleted function 'BearSSL::X509List& BearSSL::X509List::operator=(const BearSSL::X509List&)'
```

**Comment by JiriBilek** [comment]
Please try the branch [Core_3 ](https://github.com/JiriBilek/ESP_ATMod/tree/Core_3)and let me know if the problem persists.

**Comment by JiriBilek** [comment]
Resolved by Core_3 branch and also #31
